scout/server/blueprints/variant/controllers.py

```diff
@@ -456,7 +456,7 @@ def get_loqusdb_obs_cases(store, variant_obj, category, obs_families=[]):
     return obs_cases
 
 
-def observations(store, loqusdb, case_obj, variant_obj):
+def observations(store, loqusdb, variant_obj):
```

Comment:
Perhaps type hint since we are here anyway?

Reply:
Good idea. I've also fixed the comment. It was still relative to when scout only supported one instance of loqusdb